bigtable/snippets/src/main/java/com/example/bigtable/Filters.java

```diff
@@ -0,0 +1,360 @@
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigtable;
+
+import static com.google.cloud.bigtable.data.v2.models.Filters.FILTERS;
+// [START bigtable_filters_limit_row_sample]
+// [START bigtable_filters_limit_row_regex]
+// [START bigtable_filters_limit_cells_per_col]
+// [START bigtable_filters_limit_cells_per_row]
+// [START bigtable_filters_limit_cells_per_row_offset]
+// [START bigtable_filters_limit_col_family_regex]
+// [START bigtable_filters_limit_col_qualifier_regex]
+// [START bigtable_filters_limit_col_range]
+// [START bigtable_filters_limit_value_range]
+// [START bigtable_filters_limit_value_regex]
+// [START bigtable_filters_limit_timestamp_range]
+// [START bigtable_filters_limit_block_all]
+// [START bigtable_filters_limit_pass_all]
+// [START bigtable_filters_modify_strip_value]
+// [START bigtable_filters_modify_apply_label]
+// [START bigtable_filters_composing_chain]
+// [START bigtable_filters_composing_interleave]
+// [START bigtable_filters_composing_condition]
+import com.google.api.gax.rpc.ServerStream;
+import com.google.cloud.bigtable.data.v2.BigtableDataClient;
+import com.google.cloud.bigtable.data.v2.models.Filters.Filter;
+import com.google.cloud.bigtable.data.v2.models.Query;
+import com.google.cloud.bigtable.data.v2.models.Row;
+import com.google.cloud.bigtable.data.v2.models.RowCell;
+
+public class Filters {
+  // [END bigtable_filters_limit_row_sample]
+  // [END bigtable_filters_limit_row_regex]
+  // [END bigtable_filters_limit_cells_per_col]
+  // [END bigtable_filters_limit_cells_per_row]
+  // [END bigtable_filters_limit_cells_per_row_offset]
+  // [END bigtable_filters_limit_col_family_regex]
+  // [END bigtable_filters_limit_col_qualifier_regex]
+  // [END bigtable_filters_limit_col_range]
+  // [END bigtable_filters_limit_value_range]
+  // [END bigtable_filters_limit_value_regex]
+  // [END bigtable_filters_limit_timestamp_range]
+  // [END bigtable_filters_limit_block_all]
+  // [END bigtable_filters_limit_pass_all]
+  // [END bigtable_filters_modify_strip_value]
+  // [END bigtable_filters_modify_apply_label]
+  // [END bigtable_filters_composing_chain]
+  // [END bigtable_filters_composing_interleave]
+  // [END bigtable_filters_composing_condition]
+
+  // [START bigtable_filters_limit_row_sample]
+  public static void filterLimitRowSample(String projectId, String instanceId, String tableId) {
+    // String projectId = "my-project-id";
+    // String instanceId = "my-instance-id";
+    // String tableId = "mobile-time-series";
+
+    Filter filter = FILTERS.key().sample(.5);
```

Comment:
it might be nice to have a comment representing each filter does (so if someone does reach it from Github it's easier to understand).